java-checks/src/main/java/org/sonar/java/checks/BadConstantName_S00115_Check.java

```diff
@@ -99,7 +100,8 @@ private void checkName(VariableTree variableTree) {
   private boolean isStaticFinal(VariableTree variableTree) {
     boolean isStatic = false;
     boolean isFinal = false;
-    for (Modifier modifier : variableTree.modifiers().modifiers()) {
+    for (ModifierKeywordTree modifierKeywordTree : variableTree.modifiers().modifiers()) {
```

Comment:
False positive. One of the previous commit change the return type of `tree.modifiers().modifiers()` to `ModifierKeywordTree`